bfcfg

```diff
@@ -687,10 +720,15 @@ sys_cfg()
   sys_cfg_one_byte ${tmp_file} "ENABLE_SPCR" 32 "${SYS_ENABLE_SPCR}"
   sys_cfg_one_byte ${tmp_file} "DISABLE_PCIE" 33 "${SYS_DISABLE_PCIE}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_OPTEE" 34 "${SYS_ENABLE_OPTEE}"
+  sys_cfg_one_byte ${tmp_file} "DISABLE_TMFF" 35 "${SYS_DISABLE_TMFF}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_I2C0" 36 "${SYS_ENABLE_I2C0}"
   sys_cfg_one_byte ${tmp_file} "DISABLE_FORCE_PXE_RETRY" 37 "${SYS_DISABLE_FORCE_PXE_RETRY}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_BMC_FIELD_MODE" 38 "${SYS_ENABLE_BMC_FIELD_MODE}"
 
+  # Multi-byte variable dump (read only)
+  sys_cfg_dump_multi_byte ${tmp_file} "LARGE_ICMC_SIZE" 39 4
+  sys_cfg_dump_multi_byte ${tmp_file} "CE_THRESHOLD" 45 4
+
```

Comment:
I recommend having these calls guarded by a checkpoint on ${dump_mode}, just to improve the readability and avoid confusion with calls used to set the values. Actually it was for that reason I asked to add an extra (unused) parameter and mirror sys_cfg_one_byte but without having to deal with the "set", the 'sys_cfg_dump_multi_byte' routine would handle only the 'dump' and we'll have sort of TBD/TODO tag to deal with enhancements later on.

The bfcfg is not a perfect/clean script and had many contributors/developers adding bunch of commands. One cleaner way is to handle the 'dump' separately, but as you just pointed out we don't want to cause any regression so I suggest either solutions:

Solution #1

```
if [ ${dump_mode} -eq 1 ]; then
  # Multi-byte variable dump (read only)
  sys_cfg_dump_multi_byte ${tmp_file} "LARGE_ICMC_SIZE" 39 4
  sys_cfg_dump_multi_byte ${tmp_file} "CE_THRESHOLD" 45 4
fi
```

Solution #2

```
sys_cfg_multi_byte ${tmp_file} "LARGE_ICMC_SIZE" 39 4 "SYS_LARGE_ICMC_SIZE"
sys_cfg_multi_byte ${tmp_file} "CE_THRESHOLD" 45 4 "SYS_CE_THRESHOLD"
```

What do you think?
(This is a personal preference as I would like to make the script readable and easy for other developers to follow)